docs/playbooks/intro/macosx/provision-instance.ikt

```diff
@@ -0,0 +1,23 @@
+{{/* Input to create instance using the HyperKit instance plugin */}}
+{{/* =% instanceProvision "instance-hyperkit" true  %= */}}
+
+{{ $user := flag "user" "string" "username" | prompt "Please enter your user name:" "string" (env "USER")}}
+{{ $mobyPath := flag "moby-dirh" "string" "Moby directory" | prompt "Moby directory?" "string" (cat (env "HOME") "/projects/src/github.com/linuxkit/linuxkit" | nospace) }}
```

Comment:
what does `moby` refer to here? The name is confusing. Should this be `linuxkitPath` or `imagePath`?